core/chains/evm/config/toml/config.go

```diff
@@ -815,6 +815,7 @@ func (d *DAOracle) setFrom(f *DAOracle) {
 	if v := f.OracleType; v != nil {
 		d.OracleType = v
 	}
+
```

Comment:
Sorry if this was discussed in another thread but don't we still need the `DAChainID` in the DAOracle config?

Reply:
We decided to not introduce it in this PR, since it might be changed when we actually implement the Heuristic PR. https://chainlink-core.slack.com/archives/C07PHTC2WJC/p1729795376845589?thread_ts=1728503726.711869&cid=C07PHTC2WJC